### DISCUSSION TITLE
Maintainer suggested contributions

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#1162] [Ideas]
Had an idea on stream to get maintainers engaged on the open sauce platform. 

[Link to twitch thought](https://www.twitch.tv/bdougieyo/clip/DependableAntsyTitanTTours-XHifiLBIsik9f4j1?filter=clips&range=7d&sort=time)

1. Create GitHub to be installed on repos (with only issue access). 
2. Make that app watch for issues labeled `opensauced`
3. Show those issues in the open sauced main dashboard, plus recommend those repos.
4. Last, tweet out similar to [goodfirstissues](https://twitter.com/goodfirstissue)

**Comment by mtfoley** [comment]
posting for reference, repo related to goodfirstissues github app.

https://github.com/rajatjindal/goodfirstissue
